Segment-Mixpanel.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0.0'
+  s.dependency 'Analytics', '~> 3'
```

Comment:
No it shouldn't, my problem is that I want to use Analytics 3.2.4. Assuming all tests pass I want to be on the latest version and get off the Segment pod that is still using NSUserDefaults for the data queue.